app/src/pages/dataset/DatasetExamplePage.tsx

```diff
@@ -31,21 +31,23 @@ export function DatasetExamplePage() {
       query DatasetExamplePageQuery($exampleId: GlobalID!) {
         example: node(id: $exampleId) {
           ... on DatasetExample {
-            input
-            output
-            metadata
+            revision {
+              input
+              output
+              metadata
+            }
```

Comment:
```suggestion
            latestRevision: revision {
              input
              output
              metadata
            }
```

might make the most sense to alias here

Reply:
addressed in fd82e0840a17decf47d7a43a0ee05dcacf801bcb